js/coinsbit.js

```diff
@@ -0,0 +1,742 @@
+            //     }
+            ticker = this.safeValue (response, 'result', {});
+        }
+        return this.parseTicker (ticker, market);
+    }
+
+    async fetchTickers (symbols = undefined, params = {}) {
+        const [defaultType, request] = this.handleMarketTypeAndParams ('fetchTickers', undefined, params);
+        await this.loadMarkets ();
+        let tickers = {};
+        if (defaultType === 'spot') {
+            const response = await this.spotPublicGetTickers (request);
+            //     {
+            //          success: true,
+            //          message: "",
+            //          result: {
+            //            BTC_USDT: {
+            //               at: '1640028811',
+            //               ticker: {
+            //                 bid: '46448.49682539',
+            //                 ask: '46458.41093569',
+            //                 open: '46687.39',
+            //                 high: '47550',
+            //                 low: '45584.27',
+            //                 last: '46462.77',
+            //                 vol: '32.56275846',
+            //                 deal: '1510574.62551143',
+            //                 change: '-0'
+            //               }
+            //            },
+            //            ...
+            //          },
+            //          code: "200"
+            //     }
+            tickers = this.safeValue (response, 'result', {});
+        }
+        return this.parseTickers (tickers, undefined, params);
+    }
+
+    // This modified parseTickers method needs to exist in base, to keep the KYES & pass them through parseTickers
+    parseTickers (tickers, symbols = undefined, params = {}) {
+        const result = [];
+        const values = Object.values (tickers || []);
+        const keys = Object.keys (tickers);
+        for (let i = 0; i < values.length; i++) {
+            result.push (this.extend (this.parseTicker (values[i], undefined, keys[i]), params));
+        }
+        return this.filterByArray (result, 'symbol', symbols);
+    }
+
+    parseTicker (ticker, market = undefined, key = undefined) {
+        //            BTC_USDT: {
+        //               at: '1640028811',
+        //               ticker: {
+        //                 bid: '46448.49682539',
+        //                 ask: '46458.41093569',
+        //                 open: '46687.39',
+        //                 high: '47550',
+        //                 low: '45584.27',
+        //                 last: '46462.77',
+        //                 vol: '32.56275846',
+        //                 deal: '1510574.62551143',
+        //                 change: '-0'
+        //               }
+        //            },
+        let tickerFinal = {};
+        let timestamp = null;
+        if (market === undefined) {
+            // if arrived from parseTickers
+            market = this.market (key);
+            tickerFinal = ticker['ticker'];
```

Comment:
It has to be `tickerFinal = this.safeValue (ticker, 'ticker', {});` here.

Reply:
ah, missed that. done.